src/section_09/recipe093_working_with_browser_library.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Documentation     PROBLEM:
+...               You want to send a simple HTTP request using Browser library.
+...               DISCUSSION:
+...               Making a request from inside a page context allows use of page
+...               cookies and other headers that maybe difficult to gain without
+...               using browser. This method also detects some cross site problems that
+...               may not occur without a browser.
+...               This recipe demonstrates using keywords from Collections standard library.
+...               This recipe also demonstrates installing and using an external library.
+...               This recipe has the following external dependencies:
+...               $ pip install -U robotframework-browser
+...               $ rfbrowser init
+Library           Collections
+Library           Browser
+Force Tags        py3.7    py3.8    py3.9
+
```

Comment:
py3.9 not supported at this time - some of the requirements do not have 3.9 support